src/main/resources/avro/bdg.avdl

```diff
@@ -748,299 +748,304 @@ record Variant {
 }
 
 /**
- An enumeration that describes the allele that corresponds to a genotype.
+ Allele encodings for genotypes.
  */
-enum GenotypeAllele {
+enum Allele {
```

Comment:
Well, what is Impala?  It looks like there is a workaround (define table columns yourself and set enum columns as STRING) and a patch linked to that issue.